pkg/operator/api/userconfig/apis.go

```diff
@@ -135,6 +136,12 @@ func (api *API) GetResourceType() resource.Type {
 	return resource.APIType
 }
 
+func (api *API) IsServingExternalModel() bool {
+	_, isModelReference := yaml.ExtractAtSymbolText(api.Model)
```

Comment:
You can `return yaml.StartsWithEscapedAtSymbol(api.Model)` ([here](https://github.com/cortexlabs/yaml/blob/v2/yaml.go#L468) are all of the helpers I made, let me know if you ever want to add more). You also might want to use the method directly and remove this helper. Alternatively you can use this helper everywhere (search for "StartsWithEscapedAtSymbol"), and probably also want to add a helper that wraps `yaml.ExtractAtSymbolText()` (and replace all uses of that which are called on api.Model as well)